i18n/en/code.json

```
@@ -8,52 +8,52 @@
     "description": "homepage install button"
   },
   "home.quickstart": {
-    "message": "All About Your Modern Apps by Platform Engineering"
+    "message": "Everything You Need to Know About Modern Apps with Platform Engineering."
   },
   "home.easyshipping": {
     "message": "Dynamic Configuration Management"
   },
   "home.easyshipping.1": {
-    "message": "Manage all application operations in one place, in a unified easy way"
+    "message": "Easily manage all application operations from a unified platform."
   },
   "home.easyshipping.2": {
-    "message": "Environment-agnostic application configurations"
+    "message": "Environment-agnostic application configuration for seamless deployment."
   },
   "home.easyshipping.3": {
-    "message": "Standardized and flexible platform configurations"
+    "message": "Standardized yet flexible platform configurations tailored to your needs."
   },
   "home.easyshipping.4": {
-    "message": "Kubernetes-first, lightweight and user-friendly"
+    "message": "Kubernetes-first, lightweight, and user-friendly for modern workflows."
```

```suggestion
    "message": "Kubernetes-first, lightweight, and user-friendly."
```
Same reason as above.

---

Same as I mentioned in my previous comment.

---

OK